swim/node.go

```diff
@@ -94,6 +96,11 @@ func defaultOptions() *Options {
 		Clock: clock.New(),
 
 		MaxReverseFullSyncJobs: 5,
+
+		SelfEvict: SelfEvictOptions{
+			PingDisable: false,
```

Comment:
Not sure on the Go convention here but I would expect `DisablePing` or `PingDisabled`